### ISSUE TITLE
@ui5/webcomponents-base/dist/asset-registries/i18n.js - Infinite While Loop causing page break

### ISSUE THREAD

**Issue by codefactor** [SAP/ui5-webcomponents#1318]
**Describe the bug**
The page fails to load due to a while loop with no termination condition.

**To reproduce**
Not clear - see code examples, still investigating why this is an issue - please add a condition check for this ASAP.

**Isolated example**
// nope

**Expected behavior**

**Screenshots**

**Context**
- UI5 Web Components version
- OS/Platform: {...}
- Browser *(if relevant)*: {...}
- Other information: {...}

**Affected components** *(if known)*

**Log output / Any errors** in the console
```
	let localeId = normalizeLocale(language);
	while (!bundlesForPackage[localeId]) {
		localeId = nextFallbackLocale(localeId);
	}
```

If `language` is null for whatever reason, this causes an infinite loop at the while condition. the localeId is null, and bundlesForPackage[null] is null, and nextFallbackLocale(null) is null, ad infinitum.

This is happening in the following version:
```
"@ui5/webcomponents-base": "0.0.0-c6d8872f3",
```

**Comment by codefactor** [comment]
Maybe something like this can work, or you may even want to prevent against a cycle - since this only protects if the nextFallback returns the same as previous :
```
	let localeId = normalizeLocale(language);
	while (!bundlesForPackage[localeId]) {
		const newlocaleId = localeId = nextFallbackLocale(localeId);
		if (newlocaleId === localeId) break;
	}
```

**Comment by ilhan007** [comment]
Hello @codefactor thank you for reporting this finding, we will take a look asap.

**Comment by codefactor** [comment]
@ilhan007  - I found out why the issue is happening. There was a change in location of the following file:

```
import {getLanguage} from "@ui5/webcomponents-base/dist/LocaleProvider";
```

So I changed it to this:
```
import { getLanguage } from "@ui5/webcomponents-base/dist/config/Language";
```
And then later we had some code like this:

```
import { registerI18nBundle } from "@ui5/webcomponents-base/dist/asset-registries/i18n";
registerI18nBundle("@ui5/webcomponents", { [getLanguage()] : {} });
```
And the value returned by this version of `getLanguage()` is returning `en_US` and the value that `getLocale().getLanguage()` is returning is `en` so the localeId becomes as `en` yet the `bundleURLs` map has only `en_US` as key.

By the way - this is our little hack to prevent UI5 from having any i18n messages at all because we don't need them - and we don't want the UI5 web components to make any sort of AJAX request to pull the unnecessary bundle information.

In the future we may even provide a select few messages with our own custom translations - but for now we are just hacking it out entirely to make the performance better.
